### PR TITLE
Replace old crew symlink when `crew update`

### LINKED ISSUE #5603: crew symlink broken
image

### PR DESCRIPTION
**Pull request by supechicken**
Fixes #5603

### CODE REVIEW

**Comment by uberhacker** [comment]
@supechicken: This won't run after `crew update` if buildessential is already installed.

**Comment by supechicken** [comment]
> @supechicken: This won't run after `crew update` if buildessential is already installed.

Moved to `crew_profile_base.rb`

File: packages/buildessential.rb [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 class Buildessential < Package
   description 'A collection of tools essential to compile and build software.'
-  homepage ''
```
Comment (uberhacker) [inline packages/buildessential.rb:5]: Why was this line removed?

File: packages/crew_profile_base.rb [hunk 2]
```diff
@@ -8,6 +8,9 @@ class Crew_profile_base < Package
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  # temporary fix for crew symlink
+  FileUtils.ln_sf "../lib/crew/bin/crew", "#{CREW_PREFIX}/bin/crew" if ARGV[1].eql?('update')
```
Comment (uberhacker) [inline packages/crew_profile_base.rb:13]: This won't execute either if crew_profile_base is already installed.  Take a look at line 457 in crew.

File: packages/buildessential.rb (hunk 1 above)
Comment (satmandu) [reply packages/buildessential.rb:5]: I might have done that just now while deleting an empty space and failing. Let me fix that.

**Comment by satmandu** [comment]
Do we need to have crew_profile_base version revved to get crew to notice there is an update?

(@supechicken I think this also requires a new tag for https://github.com/chromebrew/crew-profile-base too)

File: packages/crew_profile_base.rb (hunk 2 above)
Comment (satmandu) [reply packages/crew_profile_base.rb:13]: so uprev crew_profile_base?

Comment (supechicken) [reply packages/crew_profile_base.rb:13]: I see it now, crew will read version from device.json instead of package files. Bump the version of crew profile base can make a new symlink at upgrade but not update

**Comment by uberhacker** [comment]
Let's let @saltedcoffii fix this since she got us into this mess in the first place. :-P
